ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityAdapter.java

```diff
@@ -69,9 +70,9 @@
   private UnityBannerAd bannerAd;
 
   /**
-   * A list of placement IDs that are currently loaded to prevent duplicate requests.
+   * UnityInterstitialEventAdapter instance to send events from the mMediationInterstitialListener.
    */
-  private static final HashMap<String, WeakReference<UnityAdapter>> mPlacementsInUse = new HashMap<>();
+  private UnityInterstitialEventAdapter eventAdapter;
```

Comment:
Super minor thing without much impact (so could be ignored for this one) but the reference type could be `IUnityEventAdapter` (so that lets say in the future, we want some common ground, we can just re-use the same base class)